test/integration/discovered_hosts_test.rb

```diff
@@ -147,12 +147,6 @@ def select_host_checkbox(id)
     page.find("#host_ids_#{id}").click
   end
 
-  def select_from(element_id, id)
```

Comment:
I switched to the core implementation (`select2` method), which should keep up with the select2 implementation, so it should be more reliable.